packages/sit-onyx/src/components/OnyxButton/OnyxButton.vue

```diff
@@ -0,0 +1,265 @@
+<script lang="ts" setup>
+import { computed } from "vue";
+
+export type ButtonProps = {
+  label?: string;
```

Comment:
Please make the label required, we even want a label for an icon-only-button for a11y purposes.